test/suites/docker-compose.override.yml

```diff
@@ -10,7 +10,9 @@ services:
     image: wikibase/wikibase
     volumes:
       - wikibase-config:/config
+      - wikibase-extensions:/var/www/html/extensions/extensions
       - ./LocalSettings.override.php:/var/www/html/LocalSettings.d/LocalSettings.override.php
+      - ./empty.php:/var/www/html/LocalSettings.d/90_UserDefinedExtensions.php
```

Comment:
Missing something there. Why this vs (or similar):

```
 - ./config/Extensions.php:/var/www/html/LocalSettings.d/90_UserDefinedExtensions.php
```

...and where is `empty.php`?

Reply:
`empty.php` is [here](https://github.com/wmde/wikibase-release-pipeline/blob/ead1b3ac0f786d9de19e31315d56451fe3cc1ae7/test/suites/empty.php#L1). This is the current working directory when running the tests.

I think I did not want to reference `../../deploy` in there because it felt fragile and like "crossing module borders".

Reply:
From my viewpoint the coupling to the deploy configuration for both the user running all the services together, and as the basis for testing against is mostly already the case, desirable, and on purpose. Like if there is a divergence that causes the test setup to break, that is a good reason to update the test setup or tests? That said, I'm not terribly concerned about it as I see we're already bypassing any default "config" (understanding that there is currently nothing in the default config) from the deploy directory with a volume mapping and ok leaving it as is.